book/text.md

```diff
@@ -876,7 +888,8 @@ for it.[^entity] If a word doesn't fit at the end of a line, check if
 it has soft hyphens, and if so break the word across lines. Remember
 that a word can have multiple soft hyphens in it, and make sure to
 draw a hyphen when you break a word. The word
-"super­cala­fraga­listic­expi­ala­do­shus" is a good test case.
+"super­cala­fraga­listic­expi­ala­do­shus"
+is a good test case.
```

Comment:
Dunno what this edit does but make sure you kept the unicode soft hyphens embedded in there.